src/main/java/org/cloudburstmc/server/config/ServerConfig.java

```diff
@@ -12,133 +12,6 @@
  */
 public class ServerConfig {
 
-
-    public static class Properties {
```

Comment:
I think I might as well remove the Properties facade.
because getConfig().getProperties().getServerIp() might be not so easy to discover. While getConfig().getServerIp() will compromise the hierarchy idea of the ServerConfig class, I believe this trade it worthwhile because I initially created the Properties facade just to comply with the rest